scripts/get-icons/index.ts

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+
+const ICON_FILE = 'Ai7AELHC6KCz38qKZkvuHo';
+
+const ICON_FRAME = 'UI icons 24(w)x24(w)';
+
+const OUTPUT_DIR = './svgs';
+
+const FIGMA_OPTIONS = {
+	headers: {
+		'X-Figma-Token': process.env.FIGMA_TOKEN ?? 'ADD ME!!!',
+	},
+};
+
+interface FigmaComponentsResponse {
+	meta: {
+		components: Array<{
+			containing_frame?: {
+				name: string;
+			};
+			name: string;
+			node_id: string;
+		}>;
+	};
+}
+
+interface Node {
+	name: string;
+	node_id: string;
+}
+
+interface NodeWithUrl extends Node {
+	url: string;
+}
+
+const getUrlsForNodes = (nodes: Node[]): Promise<NodeWithUrl[]> => {
+	return new Promise((resolve) => {
```

Comment:
Just to help me understand, why do we need to wrap the `axios` response in a Promise? My brain is thinking it's already returning a `Promise` 🤔  😄

Reply:
We don't 👍 Good spot. I've removed it now and it all still works as expected.